aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -207,7 +230,9 @@ Tensor add_sparse(const Tensor& self, const Tensor& other, Scalar alpha) {
   // TODO: Why?! Can't we just flip the order here...
   TORCH_CHECK(!(self.is_sparse() && !other.is_sparse()),
               "add(sparse, dense) is not supported. Use add(dense, sparse) instead.");
-  Tensor result = at::empty({0}, self.options());
+  auto commonDtype = at::result_type(self, other);
+  alpha_check(commonDtype, alpha);
```

Comment:
We now have new `alpha_check` and `sub_check` things added to sparse addition and sparse division, respectively. We should add a test (or two) that check that they work as expected (i.e., perform a sparse addition with an incorrect alpha and assert that the error message is what we expected).

Reply:
These are used for dense tensors as well, so we do have test coverage through that code path